build/build.xml

```diff
@@ -35,10 +35,10 @@
 				<exclude name="mckoi/**" />
 				<exclude name="h2/**" />
 				<exclude name="mysql/**" />
-				<exclude name="jakarta_commons/**" />
 				<exclude name="paperclips/**" />
-				<exclude name="swt/linux65/**" />
-				<exclude name="swt/macos/**" />
+				<exclude name="swt/linux/**" />
+        <exclude name="swt/linux-arm64/**" />
```

Comment:
Indentation is off but ok :)

Reply:
> Indentation is off but ok :)

Sorry, hat mir Eclipse lokal korrekt angezeigt - hatte das erst auf der Github-Webseite gesehen. Vielleicht wäre es hier sinnvoll, in Eclipse projektspezifische Einstellungen für die Einrückung zu konfigurieren, damit diese direkt mit eingecheckt werden können.

Reply:
https://editorconfig.org/ wäre hier vermutlich das Werkzeug der Wahl.

Das würde auch Platfformübergreifend für andere IDEs oder Code Editoren funktionieren.

Reply:
```suggestion
				<exclude name="swt/linux-arm64/**" />
```